src/main/java/org/webshar/hrms/models/db/Resource.java

```diff
@@ -0,0 +1,179 @@
+package org.webshar.hrms.models.db;
+
+import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+
+@Entity
+@Table(name="resource", uniqueConstraints = {@UniqueConstraint(columnNames = {"id"})})
+public class Resource
+{
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "id", nullable = false, unique = true, length = 11)
+  private Long id;
+
+  @Column(name = "guid", nullable = false, unique = true, length = 11)
+  private String guid;
```

Comment:
As suggested by Vikram in today's discussion, we can move these common fields applicable to all models /entities to common MappedSuperClass